packages/core/src/models.ts

```diff
@@ -451,6 +451,20 @@ export const models: Models = {
             [ModelClass.IMAGE]: settings.IMAGE_HYPERBOLIC_MODEL || "FLUX.1-dev",
         },
     },
+    [ModelProviderName.VENICE]: {
+        endpoint: "https://api.venice.ai/api/v1",
+        settings: {
+            stop: [],
+            maxInputTokens: 128000,
+            maxOutputTokens: 8192,
+            temperature: 0.6,
+        },
+        model: {
+            [ModelClass.SMALL]: settings.SMALL_VENICE_MODEL || "llama-3.3-70b",
+            [ModelClass.MEDIUM]: settings.MEDIUM_VENICE_MODEL || "llama-3.3-70b",
+            [ModelClass.LARGE]: settings.LARGE_VENICE_MODEL || "llama-3.1-405b",
```

Comment:
Could probably add an image model here like `flux-dev-unsencored` as the default with the option to control in the .env file.

Reply:
I originally had image generation in here but it needs more work so I figured I'd add it in a later PR. In the first commit here it's included though.